influxql/INFLUXQL.md

```diff
@@ -103,15 +96,20 @@ VALUES       WHERE        WITH         WRITE
 
 ## Literals
 
-### Numbers
+### Integers
 
-InfluxQL supports decimal integer literals and float literals.  Hex, octal, etc. are not
-currently supported.
+InfluxQL supports decimal integer literals.  Hexadecimal and octal literals are not currently supported.
 
 ```
-int_lit             = decimal_lit .
-decimal_lit         = ( "1" .. "9" ) { decimal_digit } .
-float_lit           = decimals "." decimals .
+int_lit             = digit { digit } .
```

Comment:
I don't think we should allow decimal literals to start with `0`.  I doubt we'll ever need to allow octal literals but old people like me might get confused if they see `0777` or similar.  :)